### DISCUSSION TITLE
customize gdf add to map

### DISCUSSION THREAD

**Discussion by rh-afk** [opengeos/leafmap#537] [Q&A] [answered]
Hi everyone,

is there any way to custom add_gdf since one of my gdf is point positions retrieve from geojson esri arcgis so when plotted it will be as marker. is that any way to change the size of marker, image of marker and put pop up on it same when we use with add_points_from_xy?

**Comment by giswqs** [comment, marked as answer]
The size of the marker is not customizable, but you can change the marker icon. See https://leafmap.org/notebooks/50_marker_cluster/.
